x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -209,7 +214,12 @@ export const uiQueryParams: (
 
         if (value !== undefined) {
           if (key === 'show') {
-            if (value === 'policy_response' || value === 'details' || value === 'isolate') {
+            if (
+              value === 'policy_response' ||
+              value === 'details' ||
+              value === 'activity-log' ||
```

Comment:
perhaps it does not matter - but for the most part, we been using `_` in values like this instead of `-`.

Reply:
Indeed. Let's keep it consistent with naming conventions. Snake case it is. 👍

Reply:
done 1c54aaa4dfd07c1f713778a6e8471a99f0fea777